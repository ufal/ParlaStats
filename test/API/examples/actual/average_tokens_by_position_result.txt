[{'speech.role': 'guest', 'average_tokens': 447.41302235179785}, {'speech.role': 'regular', 'average_tokens': 262.9682086742366}, {'speech.role': 'chair', 'average_tokens': 54.36446967652388}]

+-------------+--------------------+
| speech.role |   average_tokens   |
+-------------+--------------------+
|    guest    | 447.41302235179785 |
|   regular   | 262.9682086742366  |
|    chair    | 54.36446967652388  |
+-------------+--------------------+
