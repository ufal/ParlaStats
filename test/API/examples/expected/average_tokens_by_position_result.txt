[{'speech.role': 'guest', 'average_tokens': 1077.8934426229507}, {'speech.role': 'regular', 'average_tokens': 496.90713671539123}, {'speech.role': 'chair', 'average_tokens': 143.19426456984274}]

Running time 21.019287109375 ms

+-------------+--------------------+
| speech.role |   average_tokens   |
+-------------+--------------------+
|    guest    | 1077.8934426229507 |
|   regular   | 496.90713671539123 |
|    chair    | 143.19426456984274 |
+-------------+--------------------+
